eng/pipelines/coreclr/templates/build-perf-maui-apps.yml

```diff
@@ -102,12 +117,38 @@ steps:
     displayName: Build MAUI Android
     workingDirectory: $(Build.SourcesDirectory)/MauiTesting
 
+  - script: |
+      chmod -R a+r .
+      ../dotnet.sh publish -bl:MauiiOS.binlog -f net6.0-ios -c Release
```

Comment:
Can you start putting together a document of everywhere we have a hard coded TFM. That way when a new release rolls around we will know the places we need to go update for the next TFM.